modules/deck.gl/src/layers/graph.ts

```diff
@@ -332,7 +332,7 @@ const copyUpdatesIntoBuffers = ({
 }: any) => {
   const updatedBufferNames = (names: string[], {attributes = {}}: any) =>
     names.filter((name) => attributes[name]);
-  const copyUpdateIntoBuffers = (buffers: any, names: string[], update: any) => {
+  const copyUpdateIntoBuffer = (buffers: any, names: string[], update: any) => {
```

Comment:
This function copies each update into its component buffers. Seems like the new non-plural name is less accurate?

Reply:
fixed, thanks!